graphx/src/test/scala/org/apache/spark/graphx/lib/PageRankSuite.scala

```diff
@@ -115,7 +115,7 @@ class PageRankSuite extends SparkFunSuite with LocalSparkContext {
       assert(compareRanks(staticRanks, dynamicRanks) < errorTol)
 
       val parallelStaticRanks = starGraph
-        .staticParallelPersonalizedPageRank(Array(0), numIter, resetProb).mapVertices {
+        .staticParallelPersonalizedPageRank(Array((0, 1.0)), numIter, resetProb).mapVertices {
```

Comment:
Rather than modify the existing tests, is it possible to create a new test that uses initial values that aren't 1? that would help verify the behavior is correct.

Reply:
I thought logically it would be the same. So should I still create a separate test for it?

Reply:
I suppose you aren't testing that the implementation correctly passes the initial scores and uses them, nor testing that the original method that causes 1 to be the default works now (unless some tests still cover this). There's no reason to make elaborate tests, but is there any simple test case you can copy/paste that shows the result is different and basically correct with initial scores that aren't 1?

Reply:
@EshwarSR if you can add any minimal test of initial scores that aren't 1, and probably leave the existing tests, this can be merged

Reply:
Hi @srowen I just tried comparing the scores b/w networkx and our new implementation. The scores are not aligned. Hence I was debugging if there is any issue with the code. Do you observer any place there is an error?

Reply:
I don't know this code much at all. Yeah that's the kind of thing that's important to test -- do scores that aren't 1 work as intended?

Reply:
I tried running the old implementation with hardcoded 1.0 as initial scores. Even the scores from that and networkx dont match. I'm sort of confused what is going wrong. I guess I need to see the networkx implementation to understand the exact difference between the implementations.